src/api/entities/Proposal/index.ts

```diff
@@ -207,4 +209,37 @@ export class Proposal extends Entity<UniqueIdentifiers> {
 
     return ProposalStage.Ended;
   }
+
+  /**
+   * Retrieve the minimum amount of POLYX that must be bonded by aye votes for the proposal to be considered valid
+   */
+  public async minimumBondedAmount(): Promise<BigNumber> {
+    const {
+      context: {
+        polymeshApi: {
+          query: { pips },
+          rpc: { chain },
+        },
+      },
+      pipId,
+    } = this;
+
+    const [stage, metadata] = await Promise.all([this.getStage(), pips.proposalMetadata(pipId)]);
+
+    const { end: endBlock } = metadata.unwrap();
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const opts: any = {
```

Comment:
The type here should be `{ args: []; blockHash?: string | BlockHash }`. Please modify the signature of `requestAtBlock` so that blockHash is typed as `blockHash?: string | BlockHash`